src/icons/GreenLoadingSpinner.tsx

```diff
@@ -0,0 +1,16 @@
+import LottieView from 'lottie-react-native'
```

Comment:
i guess we'll be adding more lottie animations in the near future, and this icons folder is massive already (i personally find it very hard to find the icon that i need / verify that something i need already exists in this folder) - what do you think about creating a new folder for the lottie animation files? at least then the json files won't also live in this folder, i think then you will also be able to reuse the same name for the lottie file as the react file :)